lib/ansible/modules/system/lvol.py

```diff
@@ -188,6 +192,19 @@
     lv: test
     size: 512g
     active: false
+
+# Create a thin pool of 512g.
```

Comment:
Please require as 
```
- name: Create a thin pool of 512g
  lvol:
...

```

No comment, use `name:`.

Reply:
I can fix this post merge if Shippable is clean

Reply:
I can add a patch for that... is there anything I can do to bring this forward?

Reply:
If you could raise a PR to update the examples and add `@gundalow` as a comment on the PR so I see it ill get that merged.